examples/offline_signing/3_send_message.rs

```diff
@@ -22,6 +27,25 @@ async fn main() -> Result<()> {
 
     let signed_transaction_payload = read_signed_transaction_from_file(SIGNED_TRANSACTION_FILE_NAME)?;
 
+    // TODO @thibault-martinez: I don't like that we have to refetch the prepared transaction. Will revisit later.
```

Comment:
Yeah that's annoying, maybe store the input_signing_data_entries separated and the build the prepared PreparedTransactionData from the signed transaction essence with it together

Reply:
Do you mind if I let it like that temporarily ? I'd like to solve it another way in a future PR.